README.md

```diff
@@ -60,6 +60,7 @@ $ helm search bitnami
 - [nginx-ingress-controller](https://github.com/bitnami/charts/tree/master/bitnami/nginx-ingress-controller)
 - [NodeJS](https://github.com/bitnami/charts/tree/master/bitnami/node)
 - [PyTorch](https://github.com/bitnami/charts/tree/master/bitnami/pytorch)
+- [Apache Spark](https://github.com/bitnami/charts/tree/master/bitnami/spark)
```

Comment:
This list is following alphabetic order..

Reply:
Oh, I put `Spark` first and then changed to `Apache Spark` and forgot to change the position.